### PR TITLE
Update python docs

### PR DESCRIPTION
**Pull request by sgpearse**
This PR adds API documentation to our website for vapor_utils.py and vapor_wrf.py.  It also adds a video tutorial on how to utilize these modules and the Python engine.

The changes in this PR are best viewed by comparing [the old website's documentation](https://vapor.readthedocs.io/en/latest/usage/ancillaryTools.html#python-engine), with the [proposed changes](https://vapor.readthedocs.io/en/updatepythondocs/usage/creatingNewVariablesWithPython.html).

Note: The tutorial's instructions are still valid, despite the missing "vapor" module noted in PR #2963.

### CODE REVIEW

**Comment by clyne** [comment]
@sgpearse the preview directory linked above seems to be broken. Also, what is the process for building the documentation locally? The instructions [here](https://vapor.readthedocs.io/en/readthedocs/contributing.html#contributing-to-vapor-s-documentation) seem to be out of date. For one thing, there is no 'sphynx' directory as described in the doc.

File: docs/usage/creatingNewVariablesWithPython.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. _creatingNewVariablesWithPython:
+
+Creating new variables with python
+----------------------------------
+
+Vapor allows users to create new variables based on the data that exist in their files.  Users need to select input variables that will be read in their script, and they will need to define an output variable.  If the script successfully run by the Python Engine, the output variable will be usable in the same way as the native variables are in the dataset.
```
Comment (clyne) [inline docs/usage/creatingNewVariablesWithPython.rst:6]:
```suggestion
Vapor allows users to create new variables based on the data that already exist in their files.  Users need to select input variables that will be accessed by their script, and they will need to define one or more output variables.  If the script is successfully run by the Python Engine, the output variable(s) will be usable in the same way as the native variables are in the dataset.
```

File: docs/usage/creatingNewVariablesWithPython.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Vapor allows users to create new variables to visualize with customizable Python scripts.  These variables are derived from data that is currently loaded into Vapor.  Additionally, Vapor comes with a set of convenience functions that allow users to easily compute new variables.
```
Comment (clyne) [inline docs/usage/creatingNewVariablesWithPython.rst:8]: These first two sentences just repeat what was said in the previous paragraph, don't they?

**Review by clyne — COMMENTED** [review]
@sgpearse there are no context diffs shown for the changes made to the vapor_utils.py and vapor_wrf.py files. I'm guessing that this is because this branch is being merged into 'readTheDocs', not 'main'. Should the Python files be merged into 'readTheDocs', or should they be on a separate PR?

File: docs/usage/creatingNewVariablesWithPython.rst (hunk 2 above)
Comment (sgpearse) [reply docs/usage/creatingNewVariablesWithPython.rst]: True.  Thanks and fixed.

**Comment by sgpearse** [comment]
> @sgpearse there are no context diffs shown for the changes made to the vapor_utils.py and vapor_wrf.py files. I'm guessing that this is because this branch is being merged into 'readTheDocs', not 'main'. Should the Python files be merged into 'readTheDocs', or should they be on a separate PR?

Those got added by accident.  Thanks, and they're now removed.

**Comment by clyne** [comment]
@sgpearse the PR description says that "This PR adds API documentation to our website for vapor_utils.py and vapor_wrf.py". I see two headings for these topics, but no actual documentation.

**Comment by sgpearse** [comment]
@clyne The documentation in this PR referred to the addition of "docstrings" to vapor_utils.py and vapor_wrf.py.  You noted that the changes shouldn't be on this branch, which is correct.  Changes to those files have been deleted.

I'll submit another PR targeting main, which adds these docstring documentation items.

**Review by clyne — APPROVED** [review]

**Comment by clyne** [comment]
@sgpearse Looks like there are conflicts that need to be resolved before merging.

**Comment by sgpearse** [comment]
@clyne Now fixed.

**Pull request merged**